lightbeam/api.py

```diff
@@ -331,33 +331,52 @@ async def load_descriptors_values(self):
     # }
     # (The first element is a required attribute of the assessmentItem; the other two are required elements
     # of the required nested assessmentReference.)
-    def get_params_for_endpoint(self, endpoint):
+    def get_params_for_endpoint(self, endpoint, type='required'):
```

Comment:
the default `type` here ensures non-breaking functionality for any other lightbeam functions that use this function